### PR TITLE
Update tikv-configuration-file.md

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Added `raft-client-queue-size` and `cmd-batch`

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/6876
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -120,6 +120,11 @@ This document only describes the parameters that are not included in command-lin
 + Default value: `"1s"`
 + Minimum value: `0`
 
+### `raft-client-queue-size`
+
++ Specifies the queue size of the Raft messages. If too many messages not sent in time result in a full buffer, or messages discarded, you can specify a greater value to increase stability of the system.
```
Comment (TomShawn) [inline tikv-configuration-file.md:125]:
```suggestion
+ Specifies the queue size of the Raft messages in TiKV. If too many messages not sent in time result in a full buffer, or messages discarded, you can specify a greater value to improve system stability. 
```

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -120,6 +120,11 @@ This document only describes the parameters that are not included in command-lin
 + Default value: `"1s"`
 + Minimum value: `0`
 
+### `raft-client-queue-size`
+
++ Specifies the queue size of the Raft messages. If too many messages not sent in time result in a full buffer, or messages discarded, you can specify a greater value to increase stability of the system. 
++ Default value: 8192
```
Comment (TomShawn) [inline tikv-configuration-file.md:126]:
```suggestion
+ Default value: `8192`
```

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -637,6 +642,11 @@ Configuration items related to Raftstore
 + Default value: `1`
 + Minimum value: greater than `0`
 
+### `cmd-batch`
+
++ Controls whether to enable batch processing of the requests. It can significantly improve the write performance. 
++ Default value: true
```
Comment (TomShawn) [inline tikv-configuration-file.md:648]:
```suggestion
+ Default value: `true`
```

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -637,6 +642,11 @@ Configuration items related to Raftstore
 + Default value: `1`
 + Minimum value: greater than `0`
 
+### `cmd-batch`
+
++ Controls whether to enable batch processing of the requests. It can significantly improve the write performance.
```
Comment (TomShawn) [inline tikv-configuration-file.md:647]:
```suggestion
+ Controls whether to enable batch processing of the requests. When it is enabled, the write performance is significantly improved. 
```

**Review by TomShawn — APPROVED** [review]
Some minor comment~
Rest LGTM

**Review by gengliqi — APPROVED** [review]

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**